CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Versions
 
+## x.x.x
+* Enhance banner and MREC (<AdView/>) preloading to support preloading multiple <AdView/> instances.
```

Comment:
Should reference `<AdView/>` consistently in changelog, like in 8.0.5, we used backticks.